mottak/src/main/kotlin/no/nav/melosys/soknadmottak/mottak/MottakService.kt

```diff
@@ -91,17 +92,25 @@ class MottakService(
     fun lagreDokumentOgVedlegg(
```

Comment:
Slike metoder kan godt være private

Reply:
Oops.

Reply:
Nå kom jeg på hvorfor denne ikke var private, fra https://docs.spring.io/spring-framework/docs/current/reference/html/data-access.html#transaction-declarative-annotations:

> When you use proxies, you should apply the `@Transactional` annotation only to methods with public visibility. If you do annotate protected, private or package-visible methods with the `@Transactional` annotation, no error is raised, but the annotated method does not exhibit the configured transactional settings. If you need to annotate non-public methods, consider using AspectJ (described later).

Kanskje jeg burde flytta disse metodene for lagring til `SoknadService` og hatt en avhengighet til `DokumentService` der. Hva synes du?

Reply:
Jeg tror det blir ryddigere. Lagringskoden hører ikke så mye til mottakskoden egentlig. I tillegg har vi funksjonelt sett ikke en komplett søknad uten medfølgende dokumenter.